src/layout/SideMenu.tsx

```diff
@@ -1,112 +1,85 @@
+      image: 'book'
+    },
+    {
+      name: 'Notifications',
+      page: '/notifications',
+      image: 'notification'
+    },
+    {
+      name: 'My profile',
+      page: `/profile?address=${myAddress}`,
+      image: 'idcard'
+    }
+  ];
 
   return (
-      <Menu fluid vertical tabular>
-        <Menu.Item
-          name='all'
-          active={activeItem === 'all'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          <Label>{blogCount}</Label>
-          All blogs
-        </Menu.Item>
-
-        <Menu.Item
-          name='my-blogs'
-          active={activeItem === 'my-blogs'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          My blogs
-        </Menu.Item>
-
-        <Menu.Item
-          name='following-blogs'
-          active={activeItem === 'following-blogs'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          Following blogs
-        </Menu.Item>
-
-        <Menu.Item
-          name='new-blog'
-          active={activeItem === 'new-blog'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          New Blog
-        </Menu.Item>
-
-        <Menu.Item
-          name='feed'
-          active={activeItem === 'feed'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          Feed
-        </Menu.Item>
-
-        <Menu.Item
-          name='notifications'
-          active={activeItem === 'notifications'}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/${data.name}`).catch(console.log);
-          }}
-        >
-          Notification
-        </Menu.Item>
-
-        <Menu.Item
-          name={`profile/${myAddress}`}
-          active={activeItem === `profile/${myAddress}`}
-          onClick={(e, data) => {
-            handleItemClick(e, data);
-            Router.push(`/profile?address=${myAddress}`).catch(console.log);
-          }}
-        >
-          My profile
-        </Menu.Item>
-      </Menu>
+    <Menu
+        defaultSelectedKeys={[pathname || '/all']}
+        mode='inline'
+        theme='light'
+        style={{ height: '100%', borderRight: 0 }}
+    >
+      {MenuItems.map(item =>
+      <Menu.Item key={item.page} onClick={() => Router.push(item.page)}>
+        <Icon type={item.image} />
+        <span>{item.name}</span>
+      </Menu.Item>)}
+      {!collapsed &&
+        <Menu.ItemGroup className='DfFollowed' key='followed' title='FOLLOWED BLOGS'>
```

Comment:
Don't write a text in caps.
Set it as caps in CSS of `DfSideMenu--FollowedBlogs { text-transform: uppercase; }`
```suggestion
        <Menu.ItemGroup className='DfSideMenu--FollowedBlogs' key='followed' title='Followed blogs'>
```